### PR TITLE
hide dependencies from setup screens.

### PR DESCRIPTION
**Pull request by jensens**

### CODE REVIEW

File: src/plone/app/mosaic/configure.zcml [hunk 1]
```diff
@@ -14,12 +14,16 @@
...
-      description="Installs Mosaic layout support for Plone"
+      description="Installs Mosaic layout support."
       directory="profiles/default"
       provides="Products.GenericSetup.interfaces.EXTENSION"
       post_handler="plone.app.mosaic.setuphandlers.post_handler"
       />
 
+  <utility
+      factory=".setuphandlers.HiddenProfiles"
+      name="plone.app.mosaic-hiddenprofiles" />
```
Comment (hvelarde) [inline src/plone/app/mosaic/configure.zcml]: as per ZCML style guide, the ending `/>` must be on a new line.

File: src/plone/app/mosaic/setuphandlers.py [hunk 2]
```diff
@@ -26,6 +28,23 @@
 """
 
 
+@implementer(INonInstallable)
+class HiddenProfiles(object):
+
+    def getNonInstallableProfiles(self):
+        """Hide uninstall profile from site-creation and quickinstaller"""
```
Comment (hvelarde) [inline src/plone/app/mosaic/setuphandlers.py]: docstrings must en with a `.`.

Comment (jensens) [reply src/plone/app/mosaic/setuphandlers.py]: we may want to change it in https://github.com/plone/bobtemplates.plone/blob/master/bobtemplates/plone/addon/src/%2Bpackage.namespace%2B/%2Bpackage.name%2B/setuphandlers.py.bob#L10 where i copied it from ;)

File: src/plone/app/mosaic/configure.zcml (hunk 1 above)
Comment (jensens) [reply src/plone/app/mosaic/configure.zcml]: we may want to change it in https://github.com/plone/bobtemplates.plone/blob/master/bobtemplates/plone/addon/src/%2Bpackage.namespace%2B/%2Bpackage.name%2B/configure.zcml.bob#L42 where I copied it from...

**Pull request merged**